web/packages/teleterm/src/tray.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Menu, nativeImage, NativeImage, Tray } from 'electron';
+
+import { getAssetPath } from 'teleterm/mainProcess/runtimeSettings';
+import { RuntimeSettings } from 'teleterm/mainProcess/types';
+
+export function setTray(
+  runtimeSettings: RuntimeSettings,
+  window: { show(): void }
+): void {
+  const tray = new Tray(
+    getIcon(runtimeSettings),
+    // A random GUID that allows the icon to retain its position between relaunches.
+    'acf0cb59-0f9e-412a-8973-9ee803bc39f6'
```

Comment:
What happens if I run both the packaged app and the dev version at the same time? Should we use different IDs for these two versions?

Reply:
On Windows "changing the path to the executable will break the creation of the tray icon and a new GUID must be used", so it's probably safer to have separate guids.